src/provisioner/provisioner.py

```diff
@@ -195,8 +220,29 @@ def cluster_bootstrap(cortx_conf_url: str = None):
                 cmd_proc = SimpleProcess(cmd)
                 _, err, rc = cmd_proc.run()
                 if rc != 0 or err.decode('utf-8') != '':
+                    CortxProvisioner._update_deployment_status(
+                        cortx_config_store, node_id, 'deployment', 'error')
                     raise CortxProvisionerError(
                         rc, "%s phase of %s, failed. %s", interface,
                         components[comp_idx]['name'], err)
 
+                CortxProvisioner._update_deployment_status(
+                    cortx_config_store, node_id, 'deployment', 'progress')
```

Comment:
Pass global defs as parameters

Reply:
Done, defined all provisioner_stages and status values in constant file.